tests/test_migrators.py

```diff
@@ -149,6 +149,322 @@
+{% set download_url = "https://github.com/git-lfs/git-lfs/releases/download" %}
+
+package:
+  name: git-lfs
+  version: {{ version }}
+
+source:
+  url: {{ download_url }}/v{{ version }}/git-lfs-linux-amd64-{{ version }}.tar.gz  # [linux]
+  fn: git-lfs-linux-amd64-{{ version }}.tar.gz  # [linux]
+  sha256: 97e2bd8b7b4dde393eef3dd37013629dadebddefcdf27649b441659bdf4bb636  # [linux]
+
+  url: {{ download_url }}/v{{ version }}/git-lfs-darwin-amd64-{{ version }}.tar.gz  # [osx]
+  fn: git-lfs-darwin-amd64-{{ version }}.tar.gz  # [osx]
+  sha256: e41ac4988bd6bd38faf7c17562273cb57099b3650e50f66013aa36d62aa7448a  # [osx]
+
+  url: {{ download_url }}/v{{ version }}/git-lfs-windows-amd64-{{ version }}.zip  # [win]
+  fn: git-lfs-windows-amd64-{{ version }}.zip  # [win]
+  sha256: ebbab07348dbe71a5c20bfbdfafe4dbbafc8deacea6e6bf4143556721c860821  # [win]
+
+build:
+  number: 0
+
+test:
+  commands:
+    - git-lfs --help
+
+about:
+  home: https://git-lfs.github.com/
+  license: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.md'
+  summary: An open source Git extension for versioning large files
+
+extra:
+  recipe-maintainers:
+    - dfroger
+    - willirath
+'''
+
+sample_r = '''{% set version = '1.3-1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rprojroot
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: rprojroot_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/rprojroot_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/rprojroot/rprojroot_{{ version }}.tar.gz
+  sha256: 628c2c064b2b288264ecab6e670f9fd1d380b017a546926019fec301a5c82fca
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-backports
+
+  run:
+    - r-base
+    - r-backports
+
+test:
+  commands:
+    - $R -e "library('rprojroot')"  # [not win]
+    - "\"%R%\" -e \"library('rprojroot')\""  # [win]
+
+about:
+  home: https://github.com/krlmlr/rprojroot, https://krlmlr.github.io/rprojroot
+  license: GPL-3
+  summary: Robust, reliable and flexible paths to files below a project root. The 'root' of a
+    project is defined as a directory that matches a certain criterion, e.g., it contains
+    a certain regular file.
+  license_family: GPL3
+  license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'  # [unix]
+  license_file: '{{ environ["PREFIX"] }}\R\share\licenses\GPL-3'  # [win]
+
+extra:
+  recipe-maintainers:
+    - johanneskoester
+    - bgruening
+    - daler
+    - jdblischak
+    - cbrueffer
+'''
+
+updated_sample_r = '''{% set version = "1.3-2" %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rprojroot
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: rprojroot_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/rprojroot_{{ version }}.tar.gz
```

Comment:
A problem with testing R feedstocks is that this URL no longer works once a new version is released and this tarball is available at the Archive URL, but the bot only looks at the non-Archive URL to get the checksum, so this test will start failing if the version of r-rprojroot is released. Does anybody have a way to get around this?

Reply:
Check both urls? Only one of the 2 urls will work, but one of them will always work.